pkg/kopia/command/common.go

```diff
@@ -15,16 +15,34 @@
 package command
 
 import (
+	"os"
+
 	"github.com/kanisterio/kanister/pkg/field"
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/logsafe"
 )
 
+const (
+	// LogLevelVarName is the environment variable that controls Kopia log level.
+	LogLevelVarName = "KOPIA_LOG_LEVEL"
+	// FileLogLevelVarName is the environment variable that controls Kopia file log level.
+	FileLogLevelVarName = "KOPIA_FILE_LOG_LEVEL"
+)
+
+func LogLevel() string {
+	return os.Getenv(LogLevelVarName)
+}
+
+func FileLogLevel() string {
+	return os.Getenv(FileLogLevelVarName)
+}
+
```

Comment:
Hi Eugen,

First off, thanks for log flags! I've missed them recently (and hard-coded locally for tests) :).

What do you think about encapsulating the defaulting logic within envvar utility funcs?
For example, we could use a generic function like `nonEmptyOrDefault` to handle defaults more elegantly.

Here’s an example of what this might look like:

```
const (
...
    DefaultLogLevel     = "info"
    DefaultFileLogLevel = "debug"
)

func nonEmptyOrDefault[T comparable](t T, def T) T {
    var empty T
    if t != empty {
        return t
    }
    return def
}

func getEnv(name, def string) string {
    return nonEmptyOrDefault(os.Getenv(name), def)
}

func LogLevel() string {
    return getEnv(LogLevelVarName, DefaultLogLevel)
}

func FileLogLevel() string {
    return getEnv(FileLogLevelVarName, DefaultFileLogLevel)
}
```

and after we can refactor `commonArgs` as:

OLD
```
        ...
	logLevel := cmdArgs.LogLevel
	if logLevel == "" {
		logLevel = LogLevel()
	}

	if logLevel != "" {
		c = c.AppendLoggableKV(logLevelFlag, logLevel)
	} else {
		c = c.AppendLoggableKV(logLevelFlag, LogLevelError)
	}

	fileLogLevel := cmdArgs.FileLogLevel
	if fileLogLevel == "" {
		fileLogLevel = FileLogLevel()
	}

	if fileLogLevel != "" {
		c = c.AppendLoggableKV(fileLogLevelFlag, fileLogLevel)
	}
	...
```

NEW
```
        ...
	logLevel := nonEmptyOrDefault(cmdArgs.LogLevel, LogLevel())
	c = c.AppendLoggableKV(logLevelFlag, logLevel)

	fileLogLevel := nonEmptyOrDefault(cmdArgs.FileLogLevel, FileLogLevel())
	c = c.AppendLoggableKV(fileLogLevelFlag, fileLogLevel)
	...
```

Reply:
accepted, done.